library/Witch/hie.yaml

```diff
@@ -0,0 +1,6 @@
+cradle:
+  stack:
+    - path: ./src
+      component: "anf-ir"
+    - path: ./test
+      component: "anf-ir:test:anf-ir-test"
```

Comment:
Best to add this to the top one, I was told by the `HIE` guys that this was more proper.

Reply:
I don't underestand what you mean.

Reply:
I see, you want this bit moved to https://github.com/anoma/juvix/blob/develop/hie.yaml
Got it 👍